mmdet/integration/nncf/compression.py

```diff
@@ -144,11 +144,33 @@ def dummy_forward(model):
         with ctx:
             model(img)
 
+    def wrap_inputs(args, kwargs):
+        # during dummy_forward
+        if not len(kwargs):
+            return args, kwargs
+
+        # during building original graph
+        if not kwargs.get('return_loss') and kwargs.get('forward_export'):
+            return args, kwargs
+
+        # during model's forward
+        assert 'img' in kwargs
+        img = kwargs['img']
+        if isinstance(img, list):
```

Comment:
Please, add `assert len(img) == 1`

Also, please, add explanation messages for all `assert`-s in this function